### ISSUE TITLE
Delete Buttons not red in multiple places

### ISSUE THREAD

**Issue by Gavinp14** [open-learning-exchange/planet#7678]
[
<img width="308" alt="delete3" src="https://github.com/user-attachments/assets/c0ea6bc2-22ad-4abc-a6af-ee4ad04d2e49">
<img width="1255" alt="delete2" src="https://github.com/user-attachments/assets/74c8ae62-c818-41bd-9181-ef1f15dd2c82">
<img width="1261" alt="delete1" src="https://github.com/user-attachments/assets/e8d669bb-ac5a-439e-ba6b-7f7b9e99dfff">
](url)

I've attached a few instances where the delete buttons are not red. Although this is not a major issue, from a ui and ux perspective given that they are destructive actions they should be red and not accent colors which the user could possibly get confused with a different button

**Comment by Gavinp14** [comment]
I personally think that any button that says "Delete" or "Remove" in it should be red. I'm thinking action buttons should have a fill color of red while dropdown buttons like the first photo should just be outlined in red

**Comment by RheuX** [comment]
if its the dropdown one, i agree it should be highlighted red, but at that point, it will look weird higlighted on the middle, so we should just put it in the bottom, i think every destruction action should be different color coded, and be the last option

**Comment by Gavinp14** [comment]
I agree, I'm going to make a test branch for this and play around with different variations and colors for buttons both action and dropdowns and I will attach photos of some concepts later, feel free to do the same!

**Comment by sahilvunnam** [comment]
I think that every button that says "Delete" or "Remove" should be red, but I think if there is a dropdown "Delete" or "Remove" the font color should be red and so should the icon, rather than just highlighting the box red.

**Comment by Gavinp14** [comment]
Here are some concepts that I came up with for some of the delete buttons I have seen. I moved any delete button to the bottom because it makes more sense. Let me know what you guys think.
<img width="602" alt="deletefix2" src="https://github.com/user-attachments/assets/5ae63790-7a87-4ace-a5fa-1f979b4853b5">
<img width="527" alt="deletefix3" src="https://github.com/user-attachments/assets/56ed3202-63b0-4143-ae41-0744c6c9b83e">
<img width="605" alt="redDeleteButton " src="https://github.com/user-attachments/assets/d037d73f-8609-4746-9265-4efa4f26814f">

**Comment by jessewashburn** [comment]
These look good. Unrelated, but I am wondering why in the first screenshot, add members is a different color than add resources/add courses.

**Comment by Gavinp14** [comment]
That is a good point Jesse. @Mutugiii is this intentional or was it a mistake?

**Comment by Mutugiii** [comment]
Will be handled with a global themeing system change
